.storybook/main.ts

```diff
@@ -17,8 +17,9 @@ import type { StorybookConfig } from "@storybook/nextjs"
 const config: StorybookConfig = {
   stories: [
     "../src/components/**/*.stories.{ts,tsx}",
-    "../src/@chakra-ui/stories/*.stories.tsx",
+    // "../src/@chakra-ui/stories/*.stories.tsx",
```

Comment:
is there a reason why you left it commented?

Reply:
Nope! 😅